pkg/spdk/disk.go

```diff
@@ -159,13 +159,13 @@ func getDiskPath(path string) string {
 }
 
 func getDeviceNumber(filename string) (string, error) {
```

Comment:
Can we use `DetectDevice` in `go-spdk-helper/pkg/util/device.go`?

Reply:
Yes, same result. will use it instead.